create_infotexts.py

```diff
@@ -114,6 +117,111 @@ def create_smvk_mm_link(item):
     return smvk_link
 
 
+def generate_list_of_stripped_keywords(keyword_string):
+    """
+    Transform string of keywords from column <Nyckelord> to list of keywords.
+    
+    :param keyword_string: String from column <Nyckelord.
+    :return: list of keywords.
+    """
+    keywords_list = keyword_string.split(", ")
+    if "Svenska Cypernexpeditionen" in keywords_list:
+        keywords_list.remove("Svenska Cypernexpeditionen")
+
+    return keywords_list
+
+def append_keywords_to_desc(description_str, kw_list):
+    """
+    Append a list of keywords to the existing <Beskrivning> field value.
+
+    :param item: dictionary containing the metadata for an image.
+    :param kw_list: list of keywords found in column <Nyckelord>.
+    :return: string representing concatenation of old <Beskrivning> plus keywords list.
+    """
+    newdesc = description_str + "\n"
+    newdesc += "'''Nyckelord:'''\n"
+
+    for kw in kw_list:
+        newdesc += kw + "\n"
+    return newdesc
+
+
+def process_keyword_addition_to_description(description_str, keywords_list):
+    """
+    Append keywords to description, unless the only keyword is "Svenska Cypernexpeditionen". 
+    
+    :param fotonr: string representing the name of the image minus extension.
+    :param description_str: string respresenting the value of column <Beskrivning>.
+    :param keywords_str: string representing a comma-separated list of keywords.
+    :return: None (The altered description strings are returned in append_keywords_to_desc())
```

Comment:
Good one :+1: 

I realized that that approach didn't work and forgot to update the docstring.